.changes/next-release/enhancement-cloudformationgettemplatetemplatebodyordering-86054.json

```diff
@@ -0,0 +1,5 @@
+{
+  "category": "cloudformation get_template template body ordering", 
+  "type": "enhancement", 
+  "description": "fixes boto/boto3`#1378 <https://github.com/boto/botocore/issues/1378>`__"
```

Comment:
I think the link is messed up here. The url is pointing to `botocore` (which does not exist for that issue number) but really needs to point to `boto3`.